src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        final int arrayLenght = 6;
+        final FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figureArray = new Figure[arrayLenght];
+
+        for (int i = 1; i <= arrayLenght; i++) {
+            if (i <= arrayLenght / 2) {
+                figureArray[i - 1] = figureSupplier.getRandomFigure();
```

Comment:
```suggestion
        for (int i = 0; i < arrayLenght; i++) {
            if (i <= arrayLenght / 2) {
                figureArray[i] = figureSupplier.getRandomFigure();
```
it's ok as is, but you should gradually get used to iterating from zero :)